File: src/main/java/io/alpere/crudfop/audit/TemplateAuditProvider.java
package io.alpere.crudfop.audit;

import java.util.Optional;

public class TemplateAuditProvider implements AuditProvider {
    @Override
    public Optional<String> getCurrentAuditor() {
        return Optional.of("Crudfop Auditor");
    }

    @Override
    public String user() {
        return "unknown user";
    }
}

File: src/main/kotlin/io/alpere/crud/audit/TemplateAuditProvider.kt
package io.alpere.crud.audit

import java.util.*

class TemplateAuditProvider: AuditProvider {
    override fun user(): String {
        return "unknown user"
    }

    override fun getCurrentAuditor(): Optional<String> {
        return Optional.of("Crud-lib Auditor")
    }
}